src/pages/UserHome.jsx

```diff
@@ -41,12 +44,27 @@ const useStyles = makeStyles((theme) => ({
   noScholarshipsBox: {
     textAlign: 'center',
   },
+  progress: {
+    display: 'block',
+  },
+  loadMoreButton: {
+    margin: theme.spacing(3, 0),
+  },
 }));
 
 export default function UserHome() {
   const classes = useStyles();
+  const [scholarships, setUserHome] = useState([]);
+  const loading = useState(true);
+  const error = useState();
   const user = firebase.auth().currentUser;
 
+  useEffect(() => {
+    Scholarships.list({ authorId: user.uid }).then((results) =>
+      setUserHome(results)
+    );
```

Comment:
You can rewrite the pattern `(X) => someFunction(X)` as simply `someFunction`. You can rewrite this (assuming the rename from my other comment) as:
```js
    Scholarships.list({ authorId: user.uid }).then(setScholarships);
```

Reply:
This is still relevant